.github/workflows/ci_tests.yaml

```diff
@@ -0,0 +1,100 @@
+name: Continuous Integration tests
+
+on:
+  pull_request:
+  push:
+  # Run once a day
+  schedule:
+    - cron: '0 0 * * *'
+
+env:
+  AWS_ENCRYPTION_SDK_PYTHON_INTEGRATION_TEST_AWS_KMS_KEY_ID: |
+    arn:aws:kms:us-west-2:658956600833:key/b3537ef1-d8dc-4780-9f5a-55776cbb2f7f
```

Comment:
assuming these are the creds already out in the wild...they aren't in the travis file

Reply:
They're CMK ARNs not creds, but for Travis and Appveyor they're set through the global environment variable settings via the web UI.